gemfire-core/src/main/java/com/gemstone/gemfire/internal/cache/store/ManagedDirectBufferAllocator.java

```diff
@@ -79,12 +64,6 @@ public static long defaultMaxMemory() {
         .getTotalPhysicalMemorySize() << 2) / 5 + 7) >>> 3) << 3;
   }
 
-  public ManagedDirectBufferAllocator initialize() {
-    this.logger = initLogger();
-    DirectBufferAllocator.setInstance(this);
-    return this;
-  }
```

Comment:
FYI, retained this change (i.e. removal of initialize() method) and added `DirectBufferAllocator.setInstance(instance);` to its `ManagedDirectBufferAllocator.instance()` method.
The logger will get initialized anyway, being a member variable.